grain/_src/python/dataset/sources/parquet_dataset_test.py

```diff
@@ -2,8 +2,7 @@
 from absl.testing import absltest
 from grain._src.python.dataset.sources import parquet_dataset
 import grain.python as grain
-import pyarrow as pa
-import pyarrow.parquet as pq
```

Comment:
Even when wrapped in `lazy_import` gets partially imported anyway causing abort trap error for macos arm, due to TF 2.20.0rc0 and PyArrow conflict:
https://github.com/tensorflow/tensorflow/issues/98563